app/components/filters/filter.ts

```diff
@@ -23,7 +23,26 @@ export default class FilterComponent<S = Signature> extends Component<S> {
    * @returns queryParameter value. Possibly undefined
    */
   getQueryParam(param: string): string | undefined {
+    // param can be gemeentes,provincies
+    // if the queryParam is delimited by a comma, we want to split it and map through this.router.currentRoute.queryParams
+    if (param.includes(',')) {
+      const params = param.split(',');
+      const queryParams = this.router.currentRoute.queryParams;
+      const values: string[] = [];
+
+      console.log(params);
+
+      params.map((param) => {
+        if (queryParams[param]) {
+          values.push(queryParams[param] as string);
+        }
+      });
+      console.log(values);
+      return values.join(',');
+    }
```

Comment:
**Blocker**: You test if `queryParam` contains "," twice. One time here and one time in [select-multiple-filter.ts](https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/176/files#diff-b4e1d8e75f9d4227dd3d930f4b58770405fac4cd0afbcd478e8b3f7370168277) inserted. 

**Proposale**: 
Revert `getQueryParam` modifications, and manage the multiple `queryParam` only in [select-multiple-filter.ts](https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/176/files#diff-b4e1d8e75f9d4227dd3d930f4b58770405fac4cd0afbcd478e8b3f7370168277) and call `getQueryParam` for each param.